### PR TITLE
Qbri123 34 checking the compliance of the quantitative value of the filter

### PR DESCRIPTION
**Pull request by Goldbe54**
image

### CODE REVIEW

File: src/test/java/base/AbstractBaseTest.java [hunk 1]
```diff
@@ -27,18 +27,18 @@ public void openBrowser() {
...
+//        driver.quit();
+//    }
+//
+//    public static void sleep(int seconds) {
+//        try {
+//            Thread.sleep(seconds * 1000L);
+//        } catch (InterruptedException e) {
+//            e.printStackTrace();
+//        }
+//    }
```
Comment (Goldbe54) [inline src/test/java/base/AbstractBaseTest.java:41]: uncomment

File: src/test/java/base/AbstractBaseTest.java [hunk 2]
```diff
@@ -27,18 +27,18 @@ public void openBrowser() {
...
-            Thread.sleep(seconds * 1000L);
-        } catch (InterruptedException e) {
-            e.printStackTrace();
-        }
-    }
+//    @AfterMethod
+//    public void closeBrowser() {
+//        driver.quit();
+//    }
+//
```
Comment (SoniaMiach) [inline src/test/java/base/AbstractBaseTest.java]: ctrl/

File: src/test/java/pages/BrowseProductsPage.java [hunk 3]
```diff
@@ -136,4 +147,21 @@ public String getGenderField(String gender) {
...
+    public void loadMore() {
+       waitUntilElementToBeClickable(LOAD_MORE_BUTTON).click();
+    }
+
+    public WebElement loadMoreElement() {
+        return waitUntilElementToBeClickable(LOAD_MORE_BUTTON);
+    }
+
+    public void loadAll() {
+        do{
```
Comment (Goldbe54) [inline src/test/java/pages/BrowseProductsPage.java:160]: use ctrl+alt+l

Comment (Goldbe54) [inline src/test/java/pages/BrowseProductsPage.java:160]: make while loop

File: src/test/java/tests/FiltersTest.java [hunk 4]
```diff
@@ -79,6 +80,21 @@ public void silhouetteFilterTest() {
         softAssert.assertAll();
     }
 
+    @Test(description = "Checking the compliance of the quantitative value of the filter")
```
Comment (Goldbe54) [inline src/test/java/tests/FiltersTest.java:83]: rename please

File: src/test/java/tests/FiltersTest.java [hunk 5]
```diff
@@ -79,6 +80,21 @@ public void silhouetteFilterTest() {
...
+        BrowseProductsPage browseProductsPage = new BrowseProductsPage(driver);
+        SoftAssertions softAssert = new SoftAssertions();
+
+        headerFragment.openAllMenShoes();
+        filterProductValue = browseProductsPage.getFilterCounter(FILTER_VALUE);
+        browseProductsPage.selectFilterBy(FILTER_VALUE);
+        browseProductsPage.waitLoading();
+        browseProductsPage.loadAll();
+
+        softAssert.assertThat(filterProductValue).isEqualTo(browseProductsPage.getTitlesNames().size());
```
Comment (Goldbe54) [inline src/test/java/tests/FiltersTest.java:95]: remove empty line

**Review by SoniaMiach — APPROVED** [review]

**Review by KhraschukZahar — APPROVED** [review]

**Pull request merged**